ksqldb-engine/src/main/java/io/confluent/ksql/function/udaf/BaseComparableKudaf.java

```diff
@@ -26,11 +26,11 @@
 import java.util.function.Function;
 import org.apache.kafka.streams.kstream.Merger;
 
-public abstract class BaseNumberKudaf<T extends Number> extends BaseAggregateFunction<T, T, T> {
+public abstract class BaseComparableKudaf<T> extends BaseAggregateFunction<T, T, T> {
 
   private final BiFunction<T, T, T> aggregatePrimitive;
 
-  public BaseNumberKudaf(
+  public BaseComparableKudaf(
```

Comment:
should line 47 change to represent this UDAF accepts numbers and time/date parameters? It says "number" only.

If this "number" is still used, which means the class is for numbers and times, then should the class be renamed to something that reflects the intention of it? i.e. BaseNumberAndTimeTypesKudaf? Just an idea.